src/dpp/utility.cpp

```diff
@@ -550,6 +550,31 @@ namespace dpp {
 		    return "<@&" + std::to_string(id) + ">";
 		}
 
+		std::string message_url(const snowflake& guild_id, const snowflake& channel_id, const snowflake& message_id){
+			if (guild_id == snowflake(0) || channel_id == snowflake(0) || message_id == snowflake(0)) {
+				return "";
+			}
+			return url_host + "/channels/" + std::to_string(guild_id) + "/" + std::to_string(channel_id) + "/" + std::to_string(message_id);
+		}
+
+		std::string channel_url(const snowflake& guild_id, const snowflake& channel_id){
+			if (guild_id == snowflake(0) || channel_id == snowflake(0)) {
```

Comment:
here too

Reply:
you are looking an outdated file. no?

Reply:
<img width="779" alt="image" src="https://github.com/brainboxdotcc/DPP/assets/79337999/f7966647-9689-4f66-aba8-77a9f81866f9">